### DISCUSSION TITLE
how to handle postgress db?

### DISCUSSION THREAD

**Discussion by NitinNR** [chatwoot/chatwoot#1838] [General]
as i mentioned in titled how i can delete all data from database or any one of them lets say how i can manage them ?  or is it not possible ?

**Comment by sojan-official** [comment]
@NitinNR  I don't think this is a chatwoot specific question. You can find plenty of resources online to learn more about Postgres. 
if you are looking for a GUI tool to manage the postgres database. look into https://www.pgadmin.org/

**Reply by NitinNR** [reply]
i mean how i can manage all data which is stored in the chatwoot postgress?how i can interact with that database?

**Reply by sojan-official** [reply]
You can interact with the chatwoot specific data using the rails console.  
Otherwise, use any CLI / GUI tool like the one that's mentioned above.

**Comment by da411d** [comment]
I'm hosting Chatwoot in Docker via Coolify, so I just added one more service to it (`dpage/pgadmin4:latest`)


<img width="1567" height="987" alt="image" src="https://github.com/user-attachments/assets/bc82a1f9-ca84-424d-86f9-5700b5091d89" />

As for mysql user, it was complicated to find where to edit actual values, hehe
I just wanted to edit helpcenter article slug. Here it is, then you'll see familiar table ui :)

<img width="932" height="1080" alt="image" src="https://github.com/user-attachments/assets/b650bf4d-588b-4eb3-98df-9a1e037205ee" />

(yeah I see this thread is 5 years old, just maybe it will be useful for somebody in the future)
